libsolidity/codegen/CompilerContext.cpp

```diff
@@ -167,6 +167,18 @@ CompilerContext& CompilerContext::appendJump(eth::AssemblyItem::JumpType _jumpTy
 	return *this << item;
 }
 
+CompilerContext& CompilerContext::appendInvalid()
+{
+	return *this << Instruction::INVALID;
+}
+
+CompilerContext& CompilerContext::appendConditionalInvalid()
+{
+	eth::AssemblyItem falseTag = appendConditionalJump();
```

Comment:
I wounder whether it might be better to just have a single jump:
`ISZERO <continue> JUMPI INVALID continue:`

Your code does:
`<error> JUMPI <continue> JUMP error: INVALID continue:`